### PR TITLE
podman-remote create|ps

### PR DESCRIPTION
**Pull request by baude**
enable the podman-remote client to be able to create and list
pods on a remote system.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
bot, retest this please

File: pkg/adapter/pods_remote.go [hunk 1]
```diff
@@ -5,6 +5,9 @@ package adapter
 import (
 	"context"
 	"encoding/json"
+	"github.com/containers/libpod/cmd/podman/shared"
```
Comment (rhatdan) [inline pkg/adapter/pods_remote.go]: Misplaced

File: cmd/podman/pod_ps.go [hunk 2]
```diff
@@ -487,7 +489,8 @@ func getAndSortPodJSONParams(pods []*libpod.Pod, opts podPsOptions, runtime *lib
 			return nil, err
 		}
 		ctrNum := len(ctrs)
-		status, err := shared.GetPodStatus(pod)
+		//status, err := shared.GetPodStatus(pod)
```
Comment (rhatdan) [inline cmd/podman/pod_ps.go]: Why is this commented line here?

**Comment by baude** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**